src/app.tsx

```diff
@@ -56,12 +57,15 @@ export interface FolderFileInfo extends FileInfo {
 }
 
 interface FilesContextType {
-    addAlert: (title: string, variant: AlertVariant, key: string, detail?: string) => void,
+    addAlert: (title: string, variant: AlertVariant, key: string, detail?: string | React.ReactNode,
+               actionLinks?: React.ReactNode) => void,
```

Comment:
I don't think that works does it? Doesn't seem too, because then there is one argument instead of multiple and an interface cannot be an argument. In general open to making this easier. But feels out of scope now.

Reply:
Uh, what? `addAlert: (alert: Alert) => void` and calling it as `alert({ title: "foo", variant: ... })` doesn't work? Then I'm rather disappointed by types. This duplication is annoying and error prone, and most of all does not express what we actually want.

If it's too hard, then ok to ignore of course.